swagger/en/seasons.yml

```diff
@@ -0,0 +1,64 @@
+openapi: 3.0.0
+
+info:
+  version: 1.0.0
+  title: ""
+  description:  Notes:<br>
+    - Use "https://api.pubg.com/shards/{platform}" for PC players' season stats for seasons after and including "division.bro.official.2018-10".<br>
+    - Use "https://api.pubg.com/shards/{platform-region}" for Xbox, and to get season stats for PC players' prior to "division.bro.official.2018-10".<br>
+    - Any property with \"N/A\" as its description is not currently in use.
+servers:
+  - url: https://api.pubg.com/shards/{platform}
+    description: "[PC Only] Get season stats for seasons that began after October 3rd, 2018."
```

Comment:
Nice description, but these aren't actually shown on the site. We should probably put these somewhere else.